malariagen_data/veff.py

```diff
@@ -86,29 +79,7 @@ def get_ref_allele_coords(self, chrom, pos, ref):
         "vlen",
         "ref_start",
         "ref_stop",
-        "gene_id",
-        "gene_start",
-        "gene_stop",
-        "gene_strand",
-        "transcript_id",
-        "transcript_start",
-        "transcript_stop",
-        "transcript_strand",
-        "cds_id",
-        "cds_start",
-        "cds_stop",
-        "cds_strand",
-        "intron_start",
-        "intron_stop",
-        "intron_5prime_dist",
-        "intron_3prime_dist",
-        "intron_exon_5prime",
-        "intron_exon_3prime",
-        "ref_cds_start",
-        "ref_cds_stop",
-        "ref_intron_start",
-        "ref_intron_stop",
-        "ref_start_phase",
```

Comment:
We aren't using any of these in the output dataframe, so I removed them here. If we need them later we can always put them back.